proto/private/dependencies.bzl

```diff
@@ -23,11 +23,11 @@ dependencies = {
         ],
     },
     "com_github_protocolbuffers_protobuf": {
-        "sha256": "eaba1dd133ac5167e8b08bc3268b2d33c6e9f2dcb14ec0f97f3d3eed9b395863",
-        "strip_prefix": "protobuf-3.17.0",
+        "sha256": "87407cd28e7a9c95d9f61a098a53cf031109d451a7763e7dd1253abf8b4df422",
+        "strip_prefix": "protobuf-3.19.1",
         "urls": [
-            "https://mirror.bazel.build/github.com/protocolbuffers/protobuf/archive/v3.17.0.tar.gz",
-            "https://github.com/protocolbuffers/protobuf/archive/v3.17.0.tar.gz",
+            "https://mirror.bazel.build/github.com/protocolbuffers/protobuf/archive/v3.19.1.tar.gz",
+            "https://github.com/protocolbuffers/protobuf/archive/v3.19.1.tar.gz",
         ],
     },
     "com_google_protobuf_protoc_linux_aarch64": {
```

Comment:
Can you also update the pre-compiled files below?

Reply:
Updated. Can you help create mirrors for those files?

Reply:
On it.